tools/hash_benchmark.py

```diff
@@ -34,12 +34,12 @@ def run_benchmark(deno_exe, method, input_file):
         (out, _) = p.communicate()
 
         elapsed = out.split(':')[1].strip()
-        print "[{}] {}".format(alg, elapsed)
+        print("[{}] {}".format(alg, elapsed))
```

Comment:
Is there a reason for these changes?  They seem to be unrelated to the intent of this PR.

Reply:
I added those changes in order to run with python 3 with my environment. They shouldn't be invasive towards the old workflow since it's compatible with python 2 also. Should this be in a separate PR?

Reply:
@j4qfrost yes, change to Python code should be done in a separate PR.

Reply:
https://github.com/denoland/deno/pull/7779